GithubApiWithVIPER/GithubApiWithVIPERTests/GithubApiWithVIPERTests.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  GithubApiWithVIPERTests.swift
+//  GithubApiWithVIPERTests
+//
+//  Created by Yusuf Özgül on 19.09.2020.
+//  Copyright © 2020 Yusuf Özgül. All rights reserved.
+//
+
+import XCTest
+@testable import GithubApiWithVIPER
+
+/// test cases for repo search module
+class GithubApiWithVIPERTSearchModuleests: XCTestCase {
```

Comment:
typo var ama aslında test ettiğin module değil **GithubSearchPresenterTest** presenter'ı test ediyorsun. İstersen diğerlerini de test edebilirsin ama business logic barındırmadığı için çok gerek yok gibi